pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/inv/geospatial/BaseH3IndexCreator.java

```diff
@@ -83,8 +88,9 @@ public abstract class BaseH3IndexCreator implements GeoSpatialIndexCreator {
 
   int _nextDocId;
 
-  BaseH3IndexCreator(File indexDir, String columnName, H3IndexResolution resolution)
+  BaseH3IndexCreator(File indexDir, String columnName, H3IndexResolution resolution, String tableNameWithType)
```

Comment:
(minor) Put `tableNameWithType` in front of `columnName`

Reply:
Same for other constructors

Reply:
done